readme.md

```diff
@@ -53,7 +53,7 @@ $ yarn add --dev gen-env-template
 You can now use the command like so:
 
 ```
-./node_modules/.bin/gen-env-template [path to input .env]=.env [path to output .env]=template.env
+$ npx [path to input .env]=.env [path to output .env]=.env.example
```

Comment:
Let's keep this as is; and add a new section before this as `Usage via npx` - which will be 
```bash
$ npx gen-env-template [path to input .env]=.env [path to output .env]=.env.example
```

Reply:
Updated:  [docs: add usage with npx section](https://github.com/mrsauravsahu/gen-env-template/pull/70/commits/4abf22a95373c4f1404a1c7a5bece4b436a46822)